src/com/levi/model/Player.java

```diff
@@ -0,0 +1,29 @@
+package com.levi.model;
+
+/**
+ * Create new player.
+ * @version 4.0
+ * @since 2019
+ * @author Levi Rosal
+ */
+public class Player {
+    private String name;
+    private String locFile;
```

Comment:
Consider rename `locFile` to `fileLocation`

Reply:
Done

Reply:
Done, commit 2ec4b3e.